index.html

```diff
@@ -787,36 +787,154 @@ <h4>`ThingLink`</h4>
+                but also to endpoints such as <code>things/{id}</code> in <a>WoT Thing Description Directory</a>
+                exploration services.  
+                </p>
+                <p>In the HTTP protocol, authentication can generally be negotiated by
+                the HTTP server returning a "401 (Unauthorized)" response code in conjunction with
+                a `WWW-Authenticate` header that specifies the information required for
+                authentication.  
+                To gain access, the client then needs to make another request 
+                with the necessary information.
+                <a href="https://www.iana.org/assignments/http-authschemes/http-authschemes.xhtml">There 
+                are several authentication schemes registered with IANA.</a>
+                Not all of these are in wide use, some are experimental, and there is only
+                partial overlap with the schemes supported by TDs.
+                In addition, note that the <code>oauth</code> scheme in the IANA registration refers
+                to OAuth1, which is deprecated, so it should not be used.
+                The relevant OAuth2 flow, the <code>code</code> flow, instead 
+                begins with a "302 (Redirection)" to an authentication
+                server, eventually resulting in credentials (bearer tokens in the case of WoT)
+                that can be used for access.
+                To enable security bootstrapping on a wide variety of devices as well
+                as on browsers, the following constraints should be observed:
+                </p>
+                <ul>
+                <li><span class="rfc2119-assertion" id="exploration-secboot-401">
+                After initial contact with an exploration service using the URL provided
+                by an introduction mechanism, the exploration service
+                MUST reply with either an HTTP "401 (Unauthorized)" response code 
+                or and HTTP "302 (Redirection)" response code (OAuth2 only) if appropriate
+                authentication information has not been provided but access can be granted
+                when it is.</span>
+                Note that if the exploration service does not want to provide access for
+                some reason, it can ignore the request or reply with another code such as 404 or 403.
+                Also, if no authentication is required, equivalent to the 
+                <code>nosec</code> scheme, then the 
+                system can reply immediately with the requested TD as if authentication information
+                were provided.  
+                Bypassing authentication however (that is, effectively using the <code>nosec</code>
+                security scheme to provide the TD) is only
+                appropriate if the TD served as a response does not contain and cannot be used to infer 
+                Personally Identifiable Information.
```

Comment:
I was using "nosec" as a shorthand for "no authentication" but it's not really appropriate, so I *should* try to avoid it.  Your suggestion is acceptable.